llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -1447,6 +1447,23 @@ def TuneConditionalCompressedMoveFusion
 def HasConditionalMoveFusion : Predicate<"Subtarget->hasConditionalMoveFusion()">;
 def NoConditionalMoveFusion  : Predicate<"!Subtarget->hasConditionalMoveFusion()">;
 
+def TuneMIPSP8700
+    : SubtargetFeature<"mips-p8700", "RISCVProcFamily", "Others",
+                       "MIPS p8700 processor">;
+def FeatureMIPSCMov : SubtargetFeature<"xmipscmov", "HasMIPSCMov",
```

Comment:
`FeatureVendorMIPSCMov`? Thats how all the other vendor extensions look.

Reply:
Addressed in https://github.com/llvm/llvm-project/pull/121394